### PR TITLE
Prepare for v5.3

### PR DESCRIPTION
**Pull request by nchen63**
Waiting on https://github.com/palantir/tslint/issues/2778 for 5.3

[no-log]

### CODE REVIEW

**Comment by ajafff** [comment]
Should definitely wait for #2765 or #2720

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+- [bugfix] [`await-promise`](https://palantir.github.io/tslint/rules/await-promise): Consider types derived from a Promise in union types too. (#2661)
+- [bugfix] [`no-unsafe-any`](https://palantir.github.io/tslint/rules/no-unsafe-any/): Fix bug where number literal in type position was flagged as an unsafe `any`. (#2712)
+- [api] Deprecate `Lint.Utils.objectify` (#2764)
+
+Thanks to our contributors!
+- Andy Hanson
+- Klaus Meinhardt
+- Martin Probst
+- Noah Chen
+- Adi Dahiya
```
Comment (adidahiya) [inline CHANGELOG.md]: let's remove the above two lines :joy:

**Comment by ajafff** [comment]
Reminder: Before merging this PR the changelog should be updated to include the recently merged PRs.

**Comment by andy-hanson** [comment]
Should do #2789 before fixing any others.

**Pull request merged**